R/renv.R

```diff
@@ -0,0 +1,86 @@
+translateRenvToPackrat <- function(bundleDir) {
+  renv <- jsonlite::read_json(renvLockFile(bundleDir))
+  packrat <- packratFromRenv(renv)
+
+  packratPath <- packratLockFile(bundleDir)
+  dir.create(dirname(packratPath), showWarnings = FALSE)
+  write.dcf(packrat, packratPath)
+}
+
+packratFromRenv <- function(renv) {
+  meta <- packratMeta(renv)
+  packages <- packratPackages(renv$Packages)
+  rbind_fill(list(meta, packages))
+}
+
+packratMeta <- function(renv) {
+  repos <- vapply(
+    renv$R$Repositories,
+    function(x) paste0(x$Name, "=", x$URL),
+    character(1)
+  )
+  data.frame(
+    PackratFormat = NA,
```

Comment:
I'm guessing it's okay for these to be NA but you might want to set them to some "compatible" version just in case.